### PR TITLE
update zh-translation  content/zh/docs/reference/glossary/proxy.md

### PR DESCRIPTION
**Pull request by Sophy417**

### CODE REVIEW

**Comment by idealhack** [comment]
thanks! could you please send a PR of this to the master branch too if it's not translated there?

File: content/zh/docs/reference/glossary/proxy.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+aka:
+tags:
+- networking
+---
+-->
+<!--
+ In computing, a proxy is a server that acts as an intermediary for a remote
+service.
+-->
+在计算中，代理是充当远程服务中介的服务器。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/proxy.md]:
```suggestion
在计算机领域，代理指的是充当远程服务中介的服务器。
```

File: content/zh/docs/reference/glossary/proxy.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+the Kubernetes {{< glossary_tooltip term_id="service">}} concept.
+-->
+[kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) 是集群中每个节点上运行的网络代理，实现了部分 Kubernetes {{< glossary_tooltip term_id="service">}} 概念。
+
+<!--
+You can run kube-proxy as a plain userland proxy service. If your operating
+system supports it, you can instead run kube-proxy in a hybrid mode that
+achieves the same overall effect using less system resources.
+-->
+你可以将 kube-proxy 作为普通的 userland 代理服务运行。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/proxy.md]:
```suggestion
你可以将 kube-proxy 作为普通的用户态代理服务运行。
```

File: content/zh/docs/reference/glossary/proxy.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+如果你的操作系统支持，则可以在混合模式下运行 kube-proxy ，该模式使用较少的系统资源即可达到相同的总体效果。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/proxy.md]:
```suggestion
如果你的操作系统支持，则可以在混合模式下运行 kube-proxy；该模式使用较少的系统资源即可达到相同的总体效果。
```

**Comment by Sophy417** [comment]
> translated

I‘ve sent a PR of this to the master branch already.You can check it.

**Comment by Sophy417** [comment]
> thanks! could you please send a PR of this to the master branch too if it's not translated there?

I‘ve sent a PR of this to the master branch already.You can check it.

**Comment by tanjunchen** [comment]
/retitle update zh-translation  content/zh/docs/reference/glossary/proxy.md

**Comment by xichengliudui** [comment]
/lgtm
/approve
@Sophy417 thanks!

**Pull request merged**